### PR TITLE
Added glow level usage in Differences between GLES2 and GLES3

### PR DESCRIPTION
**Pull request by InfiniteAnts**
Added a section for glow for #4102

### CODE REVIEW

File: tutorials/misc/gles2_gles3_differences.rst [hunk 1]
```diff
@@ -88,6 +88,12 @@ That means that in GLES2 environments you can only set:
 
 See :ref:`doc_environment_and_post_processing` for more information.
 
+Glow
+----
+
+Many GLES2 devices only have a maximum of 8 texture units. On such devices, only the first 6 levels of glow can be used and 
+level 7 will be ignored.
```
Comment (NathanLovato) [inline tutorials/misc/gles2_gles3_differences.rst:95]:
```suggestion
Many GLES2 devices only have a maximum of eight texture units. On such devices, you
can only use the first six levels of glow; the seventh one won't do anything.
```

**Comment by NathanLovato** [comment]
Please favor the direct voice over passive. I edited your PR directly to better fit our writing guidelines: https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

**Comment by NathanLovato** [comment]
Thanks for your contribution!

**Comment by InfiniteAnts** [comment]
@NathanLovato Thanks a lot. Will keep that in mind for future.

**Pull request merged**